examples/todomvc/index.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>dodrio • TodoMVC</title>
+        <link rel="stylesheet" href="node_modules/todomvc-common/base.css">
```

Comment:
Is there perhaps an NPM CDN thing we could use to skip the `npm install` step for this?

Reply:
Good call: https://unpkg.com/